### PR TITLE
Add self hosted runners

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

File: .github/workflows/lint-code.yml [hunk 1]
```diff
@@ -41,7 +48,7 @@ jobs:
         run: prettier --check ${GITHUB_WORKSPACE}
 
   PythonBlack:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     steps:
       - uses: actions/checkout@v3
```
Comment (mirpedrol) [inline .github/workflows/lint-code.yml:53]: is 
```
with:
          node-version: 20
```
missing here?

Comment (mashehu) [reply .github/workflows/lint-code.yml:53]: yes, good catch

Comment (mirpedrol) [reply .github/workflows/lint-code.yml:53]: 🙅 😅

**Comment by mashehu** [comment]
Superseded by #2550

**Pull request closed without merging**